std_msgs/Header header
uint16 dim
string dtype
int32 template_id
bool is_new
float32 match_score
float32[] data
